### DISCUSSION TITLE
How to add edit profile in nova without pacakage

### DISCUSSION THREAD

**Discussion by shubhamserverfront** [laravel/nova-issues#6290] [Q&A] [answered]
- Laravel Version: ^10.10
- Nova Version: ^4.0
- PHP Version: ^8.2.0
- Database Driver & Version:
- Operating System and Version: ubuntu
- Browser type and version:chrome
- Reproduction Repository: https://github.com/###/###

### Description:
I want to add edit profile option without pacakage

![image](https://github.com/laravel/nova-issues/assets/155707482/7038cd6b-c024-4d05-bfb8-38dae22dfde1)

### Detailed steps to reproduce the issue on a fresh Nova installation:
I am using nova 4 and with logout i want to add edit profile option how can i add that without using any package and i didn't find from where logout is coming i only find it's route

**Comment by webard** [comment, marked as answer]
In `NovaServiceProvider` `boot` method:

```php
  Nova::userMenu(function (Request $request, Menu $menu) {
            $menu->prepend(
                MenuItem::make(
                    'My Profile',
                    "/resources/users/{$request->user()?->getKey()}"
                ),
            );

            return $menu;
        });
```

**Reply by shubhamserverfront** [reply]
Route::middleware('auth')->group(function () {
    Route::get('/profile', [ProfileController::class, 'edit'])->name('profile.edit');
});
In profile.edit  route i want(My profile)  should go how to do that ?

**Reply by webard** [reply]
You want display nova resource page under /profile link? I'm afraid you can't do that. You can use `/profile` route as redirect to `/nova/resources/users/{USER_ID}` and define in UserPolicy methods to allow users edit ourselves.

**Reply by shubhamserverfront** [reply]
![image](https://github.com/laravel/nova-issues/assets/155707482/bf1eeb50-fb7d-4c8b-953d-fd80c2ab5970)

  Nova::userMenu(function (Request $request, Menu $menu) {
            $menu->prepend(
                MenuItem::make(
                    'My Profile',
                ),
            );
            return $menu;
        });

   I want to show this page when i click on MY Profile i don't understand how to do that ,
Is their any way to do that if any can u give me code also??
